MediaManager/Plugin.MediaManager.Android/Audio/AudioPlayerImplementation.cs

```diff
@@ -0,0 +1,206 @@
+                var state = binder.GetMediaPlayerService().MediaPlayerState;
+                return GetStatusByCompatValue(state);
+            }
+            private set
+            {
+                status = value;
+                StatusChanged?.Invoke(this, new StatusChangedEventArgs(status));
+            }
+        }
+
+        private MediaPlayerServiceBinder binder;
+        public MediaPlayerServiceBinder Binder
+        {
+            get
+            {
+                return binder;
+            }
+            set
+            {
+                binder = value;
+            }
+        }
+
+        public AudioPlayerImplementation(MediaSessionManagerImplementation sessionManager)
+        {
+            _sessionManager = sessionManager;
+            applicationContext = Application.Context;
+            mediaPlayerServiceIntent = new Intent(applicationContext, typeof(MediaPlayerService));
+            mediaPlayerServiceConnection = new MediaPlayerServiceConnection(this);
+            applicationContext.BindService(mediaPlayerServiceIntent, mediaPlayerServiceConnection, Bind.AutoCreate);
+            _sessionManager.OnStatusChanged += (sender, i) => Status = GetStatusByCompatValue(i);
+            StatusChanged += (sender, args) =>
+            {
+                if (args.Status == MediaPlayerStatus.Playing)
+                    Task.Run(() => OnPlaying());
+            };
+            
+        }
+
+        public void UnbindMediaPlayerService()
+        {
+            if (isBound)
+            {
+                applicationContext.UnbindService(mediaPlayerServiceConnection);
+                isBound = false;
+            }
+        }
+
+        public async Task Pause()
+        {
+            await BinderReady();
+            await binder.GetMediaPlayerService().Pause();
+        }
+
+        public async Task Play(IMediaFile mediaFile)
+        {
+            await BinderReady();
+            await binder.GetMediaPlayerService().Play(mediaFile);
+        }
+
+        public async Task Seek(TimeSpan position)
+        {
+            await BinderReady();
+            await binder.GetMediaPlayerService().Seek(position);
+        }
+
+        public async Task Stop()
+        {
+            await BinderReady();
+            await binder.GetMediaPlayerService().Stop();
+        }
+
+        public Task Play(IEnumerable<IMediaFile> mediaFiles)
+        {
+            throw new NotImplementedException();
+        }
+
+        internal void OnServiceConnected(MediaPlayerServiceBinder serviceBinder)
+        {
+            Binder = serviceBinder;
+            isBound = true;
+
+            if (AlternateRemoteCallback != null)
+                serviceBinder.GetMediaPlayerService().AlternateRemoteCallback = AlternateRemoteCallback;
+
+            //serviceBinder.GetMediaPlayerService().CoverReloaded += (object sender, EventArgs e) => { instance.CoverReloaded?.Invoke(sender, e); };
```

Comment:
Doesn't seem like the CoverReloaded delegate is used at all, please remove all commented code regarding it.